### PR TITLE
Update constants.js, make them correct for 2023-24 academic year

### PR DESCRIPTION
**Pull request by ssocolow**

### CODE REVIEW

**Comment by ssocolow** [comment]
This will fix the add to gcal button, which right now puts your classes in your calendar but one year ago.

File: src/constants/constants.js [hunk 1]
```diff
@@ -23,16 +23,16 @@ export const REMOTE = ["hybrid", "remote", "in-person"];
...
+    END: "20221208",
   },
   Winter: {
     START: "20230103",
     END: "20230126",
   },
   Spring: {
-    START: "20230201",
-    END: "20230512",
+    START: "20230131",
```
Comment (AndrewMuh) [inline src/constants/constants.js:34]:
```suggestion
    START: "20240131",
```

File: src/constants/constants.js [hunk 2]
```diff
@@ -23,16 +23,16 @@ export const REMOTE = ["hybrid", "remote", "in-person"];
 export const LEVELS = [0, 1, 2, 3, 4];
 export const DATES = {
   Fall: {
-    START: "20220908",
-    END: "20221209",
+    START: "20230907",
+    END: "20221208",
   },
   Winter: {
```
Comment (AndrewMuh) [inline src/constants/constants.js:29]: Please fix all the dates

**Review by AndrewMuh — CHANGES_REQUESTED** [review]

File: src/constants/constants.js (hunk 2 above)
Comment (ssocolow) [reply src/constants/constants.js:29]: Should be done now

**Review by AndrewMuh — APPROVED** [review]
LGTM

File: src/constants/constants.js [hunk 3]
```diff
@@ -23,21 +23,21 @@ export const REMOTE = ["hybrid", "remote", "in-person"];
...
+    START: "20240131",
+    END: "20240510",
   },
   PREREG: {
-    SPRING: "2022-10-30",
-    WINTER: "2022-11-08",
-    FALL: "2022-08-01",
+    SPRING: "2023-10-30",
+    WINTER: "2023-11-08",
+    FALL: "2024-04-24",
```
Comment (yechs) [inline src/constants/constants.js:40]: This should be something like 2023-04-26. I can change it after merging.

**Review by yechs — APPROVED** [review]

**Pull request merged**